examples/app_prefab/src/main.xc

```diff
@@ -0,0 +1,78 @@
+// Copyright 2022-2025 XMOS LIMITED.
+// This Software is subject to the terms of the XMOS Public Licence: Version 1.
+
+#include "app_config.h"
+#include "util/mips.h"
+#include "device_pll_ctrl.h"
+
+#include "app.h"
+#include "app_common.h"
+
+#include <platform.h>
+#include <xs1.h>
+#include <xclib.h>
+#include <xscope.h>
+
+#include <stdio.h>
+#include <stdint.h>
+#include <string.h>
+#include <assert.h>
+
+
+
+unsafe{
+
+int main() {
+
+  chan c_tile_sync;
+  chan c_audio_frames;
+  
+  par {
+
+    on tile[0]: {
+      board_dac3101_init();
```

Comment:
Bit of a PITA for this PR but could use lib_board_support now for HW setup. Maybe make an issue? lib_board_support needs to be released first anyway as L71 board support is only in develop

Reply:
I'm planning to do this in a separate PR. Move all existing examples to L71 and also add an example demonstrating shutdown.

Reply:
I've ported the examples to run on the L71 board as part of https://github.com/xmos/lib_mic_array/pull/251/commits/fe6b5c0bd342263186ac48409bb4317eb9d57610. Will add the shutdown demo in a different PR.